templates/store/_media-object-snap-partial.html

```diff
@@ -10,14 +10,18 @@ <h4 class="p-media-object--snap__title">
       {{ snap.title }}
     </h4>
     <div class="p-media-object--snap__desc">
-      <p>
-        <span class="u-off-screen">Publisher: </span>{{ snap.origin }}
-        {% if snap.developer_validation and snap.developer_validation == VERIFIED_PUBLISHER %}
-        <span class="p-verified p-tooltip p-tooltip--top-center" aria-describedby="{{ snap.package_name }}-tooltip">
-                      <img src="https://assets.ubuntu.com/v1/75654c90-rosette.svg" />
-                      <span class="p-tooltip__message u-align--center" role="tooltip" id="{{ snap.package_name }}-tooltip">Verified account</span>
-                    </span>
-        {% endif %}
+        {% if snap_show_summary %}
+        <p>{{ snap.summary }}</p>
+      {% else %}
+        <p>
+          <span class="u-off-screen">Publisher: </span>{{ snap.origin }}
+          {% if snap.developer_validation and snap.developer_validation == VERIFIED_PUBLISHER %}
+          <span class="p-verified p-tooltip p-tooltip--top-center" aria-describedby="{{ snap.package_name }}-tooltip">
+                        <img src="https://assets.ubuntu.com/v1/75654c90-rosette.svg" />
+                        <span class="p-tooltip__message u-align--center" role="tooltip" id="{{ snap.package_name }}-tooltip">Verified account</span>
+                      </span>
+          {% endif %}
+      {% endif %}
       </p>
```

Comment:
`<p>` needs to go up inside the `{% endif %}`'s

Reply:
👍